src/pages/category.js

```diff
@@ -142,14 +139,14 @@ class CategoryPage extends Component {
         </>
       )
     } else {
-      const { title, search_facets } = category
+      const { title, search_facets, default_sort_order } = category
 
       return (
         <>
           <this.Head>
             <title>{ suffixWithStoreName(title) }</title>
           </this.Head>
-          <ProductListing title={title} indexName={Config.get().algoliaIndexName} facets={search_facets} />
+          <ProductListing title={title} indexName={`${Config.get().algoliaIndexName}_${default_sort_order}`} indexNameWithoutDefaultSortOrder={Config.get().algoliaIndexName} facets={search_facets} />
```

Comment:
Maybe we should abstract out `const algoliaIndexName = Config.get().algoliaIndexName`
```suggestion
          <ProductListing title={title} indexName={`${algoliaIndexName}_${default_sort_order}`} indexNameWithoutDefaultSortOrder={algoliaIndexName} facets={search_facets} />
```